verify_v3_1_0.py

```
@@ -0,0 +1,41 @@
+    page.hover("nav")
+    page.mouse.wheel(0, 1000)
+    page.wait_for_timeout(1000)
+    page.screenshot(path="sidebar_footer_v3_1_0.png")
+
+    # Navigate to Wallet
+    try:
+        page.click("nav >> text=Wallet")
+        page.wait_for_load_state("networkidle")
+        page.wait_for_timeout(3000)
+        page.screenshot(path="wallet_v3_1_0.png")
+    except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)